bears/general/LicenseCheckBear.py

```diff
@@ -0,0 +1,37 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='licensecheck',
+        output_format='regex',
+        output_regex=r'.*: .*UNKNOWN',
+        result_message='No license found.')
+class LicenseCheckBear:
+    """
+    Attempts to check the given file for a license, by searching the start
+    of the file for text belonging to various licenses.
+    """
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='devscripts',
+                                            dnf='licensecheck')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'License'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         licensecheck_lines: int=60,
+                         licensecheck_tail: int=5000):
+        """
+        :param licensecheck_lines:
+            Specify how many lines of the file header should be parsed
+            for license information.(Default: 60). Set to 0 to parse the whole
+            file (and ignore --tail).
```

Comment:
`(and ignore --tail)` here isnt contextually accurate, as `--tail` should probably be `licensecheck_tail` and 'ignore' isnt a good description.